docs/odata-protocol/odata-protocol.md

```diff
@@ -4916,12 +4916,12 @@ processing the request.
 
 If an individual change fails due to a failed dependency, it MUST be
 annotated with term `Core.DataModificationException` and SHOULD specify
-a `responseCode` of `424` ([Failed Dependency](#ResponseCode424FailedDependency)).
+a `responseCode` of [`424 Failed Dependency`](#ResponseCode424FailedDependency).
```

Comment:
```suggestion
a `responseCode` of `424` ([Failed Dependency](#ResponseCode424FailedDependency)).
```

The field `responseCode` of structured type [`DataModificationExceptionType `](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#DataModificationExceptionType) has type `Int16` and can only take the numeric value `424`, thus this unusual formatting